PluginsAndFeatures/azure-toolkit-for-intellij/.gitignore

```diff
@@ -44,3 +44,6 @@ gen-external-apklibs/
 
```

Comment:
missing package-info.java for your package.

Reply:
checked in with Sameeksha and agreed to leave this for now